cmd/crossplane/core/core.go

```diff
@@ -76,7 +76,6 @@ type startCommand struct {
 	PollInterval     time.Duration `help:"How often individual resources will be checked for drift from the desired state." default:"1m"`
 	MaxReconcileRate int           `help:"The global maximum rate per second at which resources may checked for drift from the desired state." default:"10"`
 
-	EnableCompositionRevisions bool `group:"Alpha Features:" help:"Enable support for CompositionRevisions."`
```

Comment:
For beta features I'd like to keep the feature flag in place, but move it into `group:"Beta Features:"` and have it enabled by default. That way folks can opt-out of the feature while its in beta if they so desire. This is in line with how Kubernetes feature maturity works.

Reply:
Updated the flag name and enabled it by default.